autogoal/contrib/ensemble/__init__.py

```diff
@@ -0,0 +1 @@
+from autogoal.contrib.ensemble.stacking import StackingEnsemble
```

Comment:
I think this has to be `from autogoal.contrib.ensemble._stacking ...` with the `_` in front of `stacking` ???